honeybee/boundarycondition.py

```diff
@@ -21,18 +21,40 @@ def __init__(self, sun_exposure=False, wind_exposure=False,
                 This input can also be the word 'autocalculate' to have the view
                 factor automatically calculated.  Default: 'autocalculate'.
         """
+        sun_exposure = False if sun_exposure is None else sun_exposure
```

Comment:
We don't need it until you implement `from_dict` methods and you realize that these properties are auto-assigned a `None` if they are not included in the dictionary.  In any case, we might just get rid of the `sun_exposure` attributes on the base class and only implement it on the `Outdoors` class depending what we decide in our google doc conversation.  This might simplify the situation here.

Reply:
This one no longer exists.